pkg/util/updater/updater.go

```diff
@@ -86,60 +88,64 @@ const (
 // NOTE: Update may call os.Exit.
 func (u *Updater) Update(updateConfig UpdateConfig) (updated bool, err error) {
 	if !atomic.CompareAndSwapInt32(&u.updating, 0, 1) {
-		return false, ErrAlreadyStarted
+		return false, errors.New("updating already started")
```

Comment:
Why are we replacing this with an inline error?

Reply:
I had taken out most of the code in that file and several others, until the point it wouldnt work, so i copied the one function i modified and then git reset.

most of the errors already declared will be taken out, based on my proposal.

i will change it to non-inline errors before all is said and done.